### ISSUE TITLE
OMTypes/ModuleTypes for calibration devices (Trac #2269)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2261]
The new calibration devices will be listed in the GCD file and will need types assigned. The current request from the calibration folks is separate enum values for each of the new calibration sources (POCAM, camera, acoustic sensor, ect).

Millipede and CLSim both look at the module/OM types as part of the processing to determine which keys refer to in-ice sensors vs IceTop/IceAct/Scintillation/calibration. These are a bit of nuisance to keep adding to, so maybe we should add a function to I3OMGeo and I3ModuleGeo that just returns whether the module has an in-ice PMT or not. 

The other option would be to store a bitmask with flags for InIcePMT, InIceSource, and Surface. That would require us to increment the dataclasses version, which I'd prefer to avoid unless necessary.

Migrated from https://code.icecube.wisc.edu/ticket/2269
```json
{
    "status": "closed", 
    "changetime": "2019-06-18T20:57:20", 
    "description": "The new calibration devices will be listed in the GCD file and will need types assigned. The current request from the calibration folks is separate enum values for each of the new calibration sources (POCAM, camera, acoustic sensor, ect).\n\nMillipede and CLSim both look at the module/OM types as part of the processing to determine which keys refer to in-ice sensors vs IceTop/IceAct/Scintillation/calibration. These are a bit of nuisance to keep adding to, so maybe we should add a function to I3OMGeo and I3ModuleGeo that just returns whether the module has an in-ice PMT or not. \n\nThe other option would be to store a bitmask with flags for InIcePMT, InIceSource, and Surface. That would require us to increment the dataclasses version, which I'd prefer to avoid unless necessary.", 
    "reporter": "mjl5147", 
    "cc": "summer.blot", 
    "resolution": "fixed", 
    "_ts": "1560891440817755", 
    "component": "upgrade", 
    "summary": "OMTypes/ModuleTypes for calibration devices", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2019-04-05T14:20:49", 
    "milestone": "Summer Solstice 2019", 
    "owner": "mjl5147", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20190410T15:04:12: olivas changed milestone from "Vernal Equinox 2019" to "Summer Solstice 2019"

**Comment by icecube-trac** [comment]
Trac update at 20190604T18:50:50: olivas commented:

> Owner: Please 'accept' this ticket acknowledging you're aware of its existence.

**Comment by icecube-trac** [comment]
Trac update at 20190606T13:02:20: olivas commented:

> The the owner of this ticket: Please click 'accept' so I know you're aware of its existence.  Also be aware the due date is approaching.

**Comment by icecube-trac** [comment]
Trac update at 20190607T13:03:45: olivas commented:

> Ticket owners...please click 'Modify Ticket'->accept or give some other indication you're aware this ticket exists and is assigned to you.

**Comment by icecube-trac** [comment]
Trac update at 20190608T14:39:57: olivas commented:

> To the owner of this ticket, you don't necessarily have to 'accept' the ticket.  You have options.  You can remove yourself as the owner or...ok you only have one option.  :) Please remove yourself as the owner and I'll find someone else to take it.

**Comment by icecube-trac** [comment]
Trac update at 20190610T14:18:09: mjl5147 changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20190618T20:57:20: 
* mjl5147 changed status from "accepted" to "closed"
* mjl5147 commented:

> In changeset 174066:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="174066"
> Update the enums to allow calibration and R&D devices in the upgrade GCDs. Note that we'll probably want to take Erik's suggestion and push these new devices to a separate I3OMGeoMap and I3ModuleGeoMap so we don't have to screw around too terribly much with the rest of icetray. Closes #2261
> ```


* mjl5147 changed resolution from "" to "fixed"
